.github/workflows/tests.yml

```diff
@@ -72,19 +69,17 @@ jobs:
         run: |
           python -m pip install --upgrade pip
           pip install git+https://github.com/PennyLaneAI/pennylane.git
-          pip install -r requirements-ci-legacy.txt
+          pip install -r requirements-ci.txt
           pip install wheel pytest pytest-cov pytest-mock pytest-benchmark flaky --upgrade
-          pip freeze
 
       - name: Install Plugin
         run: |
           python setup.py bdist_wheel
           pip install dist/PennyLane*.whl
-          pip freeze
 
       - name: Run tests
         run: |
-          pl-device-test --device=qiskit.basicaer --tb=short --skip-ops --shots=20000 --device-kwargs backend=qasm_simulator
+          pl-device-test --device=qiskit.basicsim --tb=short --skip-ops --shots=20000 --device-kwargs backend=basic_simulator

```

Comment:
Is there any performance degradation for `basicsim` in comparison to `basicaer`?

Reply:
`Basicaer` is deprecated in >1.0, and `basicsim` is the 1.0 replacement for it. Shouldn't be any performance degradation as far as I can tell.